src/main/java/seedu/moneygowhere/userinterface/ConsoleInterface.java

```diff
@@ -356,15 +382,50 @@ private void runCommandEditExpense(ConsoleCommandEditExpense consoleCommandEditE
 
     @SuppressWarnings("Java8ListSort")
     private void runCommandSortExpense(ConsoleCommandSortExpense commandSortExpense) {
-        ArrayList<Expense> expenses = expenseManager.getExpenses();
-        Comparator<Expense> comparator = commandSortExpense.getComparator();
-        Collections.sort(expenses,comparator);
         expenseManager.updateSortExpenses(commandSortExpense);
         printInformationalMessage(Messages.CONSOLE_MESSAGE_COMMAND_SORTED_EXPENSE_SUCCESS);
 
         LocalStorage.saveToFile(expenseManager);
     }
 
+    private void runCommandChangeCurrency(ConsoleCommandConvertCurrency consoleCommandConvertCurrency) {
```

Comment:
Is it possible to rename the function to runCommandConvertCurrency instead? It would be more in line with the codebase's naming scheme